drivers/CoreIMU/include/CoreLSM6DSOX.h

```diff
@@ -0,0 +1,86 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "drivers/I2C.h"
+
+#include "lsm6dsox_reg.h"
+
+namespace leka {
+
+class CoreLSM6DSOX
+{
+  public:
+	explicit CoreLSM6DSOX(mbed::I2C &i2c);
+
+	enum class RateXL
+	{
+		OFF	   = lsm6dsox_odr_xl_t::LSM6DSOX_XL_ODR_OFF,
+		_26Hz  = lsm6dsox_odr_xl_t::LSM6DSOX_XL_ODR_26Hz,
+		_52Hz  = lsm6dsox_odr_xl_t::LSM6DSOX_XL_ODR_52Hz,
+		_104Hz = lsm6dsox_odr_xl_t::LSM6DSOX_XL_ODR_104Hz,
+	};
+
+	enum class RangeXL
+	{
+		_2G	 = lsm6dsox_fs_xl_t::LSM6DSOX_2g,
+		_4G	 = lsm6dsox_fs_xl_t::LSM6DSOX_4g,
+		_8G	 = lsm6dsox_fs_xl_t::LSM6DSOX_8g,
+		_16G = lsm6dsox_fs_xl_t::LSM6DSOX_16g,
+	};
+
+	enum class RateGyr
```

Comment:
The ODR are the same for both gyr and accel. I would use only one and have the value handled by a switch case down the line.